openapi.yaml

```diff
@@ -12561,14 +12560,15 @@ paths:
       servers:
       - url: https://api.linode.com/v4
       summary: Object Storage Bucket Remove
-      description: >
-        Removes a single bucket. While buckets containing objects _may_ be
-        deleted by including the `force` option in the request, such operations
-        will fail if the bucket contains too many objects. The recommended
+      description: |
+        Removes a single bucket.
+
+        Bucket objects must be removed prior to removing the bucket. While buckets containing objects _may_ be
+        deleted using the [s3cmd command-line tool](/docs/products/storage/object-storage/guides/s3cmd/#delete-a-bucket), such operations
+        can fail if the bucket contains too many objects. The recommended
```

Comment:
"Bucket objects must be removed prior to removing the bucket" - This will no longer be true once bapi support is merged and the API force flag is deprecated.

Reply:
Update: after speaking to @bbiggerr the changes here only reflect  https://bits.linode.com/LinodeAPI/apinext/pull/4001 and remove reference to the force flag (prior to it being removed). With this in mind ignore the above comment :)